extensions/skins/conversation_v1/player.html

```diff
@@ -258,11 +297,13 @@ <h4 class="conversation-skin-responses-dropdown-container" ng-click="toggleShowP
       <span class=" glyphicon glyphicon-arrow-left">
       </span>
     </li>
-    <li ng-repeat="dot in dots track by $index" class="oppia-animate-progress-dot">
+    <li ng-repeat="dot in dots | sliceDots:currentDotIndex  track by $index"
+        ng-class="dot.isVisible ? 'oppia-animate-progress-dot': 'oppia-animate-progress-dot-hidden'">
```

Comment:
I realize I suggested ng-class -- but actually Angular has a built-in directive, ng-show, for this sort of thing. It might actually be nicer to use ng-show in here based on leftmostVisibleDotIndex and rightmostVisibleDotIndex as described above.